pubspec.yaml

```diff
@@ -4,7 +4,7 @@ version: 1.0.0
 homepage: https://github.com/SAP/gigya-flutter-plugin
 
 environment:
-  sdk: ">=2.18.0 <3.0.0"
+  sdk: ">=2.18.0 <4.0.0"
```

Comment:
This is to support anyone who wants to use Dart 3.0 already. (independent of the Web SDK :) )